src/ci/run.sh

```diff
@@ -139,6 +144,10 @@ else
     RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-llvm-assertions"
   fi
 
+  if [ "$DOWNLOAD_RUSTC" = 1 ]; then
```

Comment:
Why does this need to be opt-in instead of enabled by default across all test builders? What are we losing if we do that?

(Currently we do enable downloading LLVM in all/most test builders, see below with NO_DOWNLOAD_CI_LLVM).